deployment/paiLibrary/paiService/service_template_generate.py

```diff
@@ -49,7 +49,9 @@ def template_mapper(self):
         servce_conf_dict = {
             "clusterinfo": self.cluster_object_mode['clusterinfo'],
             "machineinfo": self.cluster_object_mode["machineinfo"],
-            "machinelist": self.cluster_object_mode["machinelist"]
+            "machinelist": self.cluster_object_mode["machinelist"],
+            "serviceinfo": self.service_conf,
+            "servicename": self.service_name
```

Comment:
I don't think we should add service_name here. Pls remove it.  Object name of the pod in the kubernetes cluster should be defined by owner in their script. And this will add more complex dependency to paictl. And we should keep the logic in paictl more clean.  And note, the service_name is same as the directory name in ```src```. But the objectname may be different from it. please change it.